cmd/image-builder/describeimg.go

```diff
@@ -31,6 +34,8 @@ type describeImgYAML struct {
 	BuildPipelines   []string                 `yaml:"build_pipelines"`
 	PayloadPipelines []string                 `yaml:"payload_pipelines"`
 	Packages         map[string]*packagesYAML `yaml:"packages"`
+
+	PartitionTable *disk.PartitionTable `yaml:"partition_table,omitempty"`
```

Comment:
I guess one open question is if this should be `PartitionTable` or `BasePartitionTable`

Reply:
I think plain `PartitionTable` is enough.  I think it's clear that what we're displaying is the default configuration for the image in all respects.  Packages, hostname, locales, and all other configurable properties are displayed as their _base_ values, even though they can be changed but we don't specify that it's the "base package set", do we?

This also makes me thing that an interesting idea might be to support `describe-image` with a blueprint that modifies the image and prints the description of the result.

Reply:
I like the idea of the blueprint, that is cool (will require some work in images as currently the partition table + bp is not accessible IIRC).

Reply:
Yeah, that's for the future.  Might need a lot of internal work to create a modified image configuration.